azurerm/internal/services/containers/resource_arm_kubernetes_cluster.go

```diff
@@ -287,6 +287,19 @@ func resourceArmKubernetesCluster() *schema.Resource {
 							}, true),
 							DiffSuppressFunc: suppress.CaseDifference,
 						},
+
+						"outbound_type": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Default:  string(containerservice.LoadBalancer),
+							ForceNew: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								string(containerservice.LoadBalancer),
+								string(containerservice.UserDefinedRouting),
+							}, true),
+							DiffSuppressFunc: suppress.CaseDifference,
```

Comment:
We typically don't ignore case for enum types like this, is there a reason for this?

Reply:
nope. will remove it.

Reply:
removed.

Reply:
@katbyte I removed this, can you resolve the conversation?